src/Mvc/Extensions.ApiDescription.Server/src/build/Microsoft.Extensions.ApiDescription.Server.props

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<Project>
+  <!--
+    Settings users may update as they see fit.
+  -->
+  <PropertyGroup>
```

Comment:
@glennc do you have any input on the names of these user-visible properties?

Reply:
As we discussed can we replace Retrieve with Generate in these properties.